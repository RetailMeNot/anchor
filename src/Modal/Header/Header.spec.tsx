// REACT
import * as React from 'react';
import * as renderer from 'react-test-renderer';

// STYLED COMPONENTS
/* tslint:disable no-import-side-effect*/
import 'jest-styled-components';
/* tslint:enable */

// COMPONENT
import { Modal } from '..';
const { Header } = Modal;

describe('Component: Modal.Header', () => {
    it('should render with title', () => {
        const subject = <Header title="Header Title" />;
        const tree = renderer.create(subject).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
