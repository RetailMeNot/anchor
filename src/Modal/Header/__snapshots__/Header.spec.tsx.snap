// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Modal.Header should render with title 1`] = `
.c0 {
  box-sizing: border-box;
  min-height: 4rem;
  padding: 1.25rem 4.75rem 1.25rem 2rem;
  width: 100%;
  border-radius: 8px 8px 0 0;
  margin-bottom: 1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-order: -1;
  -ms-flex-order: -1;
  order: -1;
  color: #595959;
  background-color: #FFFFFF;
}

.c0 ~ .c2.c2 {
  padding-top: 0;
}

.c1 {
  box-sizing: border-box;
  font-family: Avenir Next,Segoe UI,Helvetica Neue,Helvetica,Roboto,sans-serif;
  font-size: 1.25rem;
  line-height: 1.5rem;
  font-weight: normal;
  text-align: inherit;
  color: inherit;
  font-weight: bold;
}

.c1 small {
  font-size: 87.5%;
}

<div
  className="c0 anchor-modal-header"
>
  <span
    className="c1 anchor-typography"
    scale={20}
  >
    Header Title
  </span>
</div>
`;
